content/en/docs/advanced-tasks/overriding-catalog-source-pod-scheduling-configuration.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Overriding CatalogSource Pod Scheduling Configuration"
+linkTitle: "CatalogSource Pod Scheduling Overrides"
+weight: 3
+---
+
+## Overriding CatalogSource Pod Scheduling Configuration
+
+When given a CatalogSource of type `grpc` with `spec.image` defined, the `catalog` operator will create 
+Pod that serves the content in `spec.image`. By default, this pod defines in its spec no `tolerations`, no `priorityClassName`,
+and only the following node selector: `kubernetes.io/os=linux`. These values can be overriden with CatalogSource's
+`spec.grpcPodConfig`. For instance,
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: operatorhubio-catalog
+  namespace: olm
+spec:
+  sourceType: grpc
+  image: quay.io/operatorhubio/catalog:latest
+  displayName: Community Operators
+  publisher: OperatorHub.io
+
+  # optional
+  grpcPodConfig:
+    # override nodeSelector
+    # for more information on nodeSelectors see:
+    # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
+    # optional
+    nodeSelector:
+      custom_label: value
+    
+    # change priorityClassName
+    # kubernetes ships with: system-cluster-critical and system-node-critical
+    # setting it to empty ("") will assign the pod the default priority.
+    # Other priority classes can be defined manually. For more information on priority classes see:
+    # https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass
+    # optional
+    priorityClassName: system-cluster-critical
```

Comment:
Any value in documenting the precedence between the annotation and the spec.grpcPodConfig, or do we want to avoid shining light on that way of configuring priority classes for the underlying pods?

Reply:
Lol - knew I forgot something